### ISSUE TITLE
Maximal number of output points

### ISSUE THREAD

**Issue by PavelBal** [Open-Systems-Pharmacology/OSPSuite.SimModel#129]
The number of output points (in the OutputSchema) is restricted to 5000. This value is a great limitation when simulating months (or even years). I think this value should be increased (or even exposed as an input parameter).

**Comment by msevestre** [comment]
This value is set in core not in simmodel I believe

**Comment by Yuri05** [comment]
Yes, it's in core. SimModel does not have any restrictions regarding the number of points

https://github.com/Open-Systems-Pharmacology/OSPSuite.Core/blob/23c54c7acc30c1910891c759443016780a12c527/src/OSPSuite.Core/Serialization/SimModel/DTO/OutputSchemaExport.cs#L25-L36
